portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,19 +37,51 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    
-  private List<String> names;
- 
-  @Override
-  public void init() {
-    names = new ArrayList<String>(Arrays.asList("Sam", "Hadley", "Jerry"));
-  }
+  
+  /** Java object converter. */
```

Comment:
I'm not sure we need comments here and below. Comments should be used only when it is infeasible to make your code self-explanatory.